src/components/Calculator.js

```diff
@@ -1,72 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
-// import PropTypes from 'prop-types';
 import Buttons from './Buttons';
 import Results from './Results';
 import calculate from '../logics/calculator';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
+export default function Calculator() {
```

Comment:
**Optional**
- Well done refactoring the component into a functional component. However, it'd be best practices if you used the arrow function ES6 feature 🙏